packages/@okta/vuepress-site/docs/reference/hook-best-practices/index.md

```diff
@@ -0,0 +1,67 @@
+
+>**Note:** This is a large list of IP Addresses and the list is subject to change. Unless required by your organization, securing your Hook by Authentication header is recommended.
+
+## Protecting your Hook content from external viewers
+
+To prevent unauthorized parties from reading the contents of an Okta Hook, we recommend the following:
+
+* Okta uses HTTPS to encrypt communications to your Hook endpoint. When using HTTPS, ensure you keep your SSL certificate updated and the Domain Name System (DNS) secured, so that someone can’t re-point your endpoint to another location.
+
+## Avoiding delays in Inline Hook responses
+
+When Okta uses an Inline Hook to communicate with your endpoint, the user experience is paused until your code responds. To prevent unnecessary delays or timeouts, we recommend the following:
+
+* Treat Okta Hooks as you would any other HTTP request, namely: Ensure you respond to the HTTP request in under 400 milliseconds with a 200 (Success) or 204 (Success no content) code.
+
+>**Note:** The Okta request times out after 3 seconds with no response for Inline Hooks and Event Hooks.
+
+See Inline Hook [Timeout and Retry](/docs/concepts/inline-hooks/#timeout-and-retry) for further information on Inline Hook timeout and retry behavior; see Event Hook [Timeout and Retry](https://developer.okta.com/docs/concepts/event-hooks/) for the same.
+
+## Working with Event Types (?)
+
+* Do we have any guidelines on the number of Event Types we can add to a hook? Our current documentation suggests we can configure "multiple event types". Would it be better to have multiple endpoints for multiple event types, from a load perspective? Or does it matter?
+
+## Managing scale of Okta Hook calls (?)
+
+* Any advice necessary on this subject, which I've seen on other webhhook pages. E.g. "If you’re on the other side running a system that is consuming webhooks, you can scale your webhook ingestion the same way you’d horizontally scale for regular web traffic — by using a load balancer or reverse-proxy in front of your web servers. "
+
+## Ordering and Duplicates of Hooks (?)
+
+* Not sure if there is anything to discuss here?
+
+## Creating error messages (?)
+
+* Any guidance for creating error messages? I can link [Inline Hooks concepts - error](/docs/concepts/inline-hooks/#error)
+
+## Troubleshooting your Hook implementations
+
+Developers and administrators can preview sample Okta calls and responses from your external service for certain Inline Hooks and Event Hooks, as well as review the Admin Console System Log to troubleshoot your implementations, in addition to your external service's logging features. See [Troubleshooting hook implementations](/docs/guides/common-hook-set-up-steps/nodejs/troubleshooting/) for further information.
```

Comment:
Preview is available for ALL event hooks and CERTAIN Inline Hooks right now so let's try to make that a bit more clear. Maybe ".....for Event Hooks and certain Inline Hooks (SAML and Registration)"